ui/src/app/shared/services/index.ts

```diff
@@ -12,8 +12,10 @@ import {RepoCredsService} from './repocreds-service';
 import {UserService} from './user-service';
 import {VersionService} from './version-service';
 import {ViewPreferencesService} from './view-preferences-service';
+// import {AppSetViewPreferencesService} from './view-preferences-service';
 export interface Services {
     applications: ApplicationsService;
+    // applicationSets: ApplicationSetsService;
```

Comment:
After looking at the changes in the ApplicationService, perhaps your comment here indicates that it might be better to have an ApplicationSetsService.  Seems like it is cleaner that way, most of the functions are short anyway, and it's probably ok to separate the concerns.